layouts/partials/fragments/list.html

```diff
@@ -16,6 +14,18 @@
       {{- $self.page_scratch.Add "pages" .Pages -}}
     {{- end -}}
   {{- end -}}
+{{- else -}}
+  {{- $page := .Site.GetPage "Section" (.Params.section | default .root.Section) -}}
+  {{- .page_scratch.Set "pages" $page.Pages -}}
+  {{- .page_scratch.Add "pages" $page.Sections -}}
+  {{- if (ne .Params.subsections false) -}}
+    {{- .page_scratch.Add "pages" $page.Sections -}}
```

Comment:
Why the double `.page_scratch.Add "pages" $page.Sections`? Or am I missing something?

Reply:
That's a bug. Line 20 should not be there.